lib/Raven/Client.php

```diff
@@ -328,7 +328,7 @@ private static function _convertPath($value)
         // we need app_path to have a trailing slash otherwise
         // base path detection becomes complex if the same
         // prefix is matched
-        if ($path{0} === DIRECTORY_SEPARATOR && substr($path, -1) !== DIRECTORY_SEPARATOR) {
+        if ($path[0] === DIRECTORY_SEPARATOR && substr($path, -1) !== DIRECTORY_SEPARATOR) {
```

Comment:
This may generate a notice with an empty string: https://3v4l.org/GVBHT

Maybe we should use `substr` here too?

Reply:
https://3v4l.org/u8HWDh - the same problem with braces